walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -40,19 +40,20 @@
    * @return Collection<TimeRange> of available TimeRange's for the meeting to be scheduled
    */
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    Collection<TimeRange> result;
+    Collection<TimeRange> result = new ArrayList<TimeRange>();
 
-    ArrayList<TimeRange> conflicts = generateConflicts(events, request.getAttendees());
+    ArrayList<TimeRange> mandatoryConflicts = generateConflicts(events, request.getAttendees());
+    ArrayList<TimeRange> optionalConflicts =
+        generateConflicts(events, request.getOptionalAttendees());
+    ArrayList<TimeRange> allConflicts = (ArrayList<TimeRange>) mandatoryConflicts.clone();
```

Comment:
I'm not a guru when it comes to Java style ( we use immutable data structures almost all of the time internally), I will say this just looks weird to me.

I think it might be easier to read if you just start with an empty list and addAll twice.